test/integration/basic_test.go

```diff
@@ -177,15 +260,22 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`Minishift should have a valid IP address`, m.shouldHaveAValidIPAddress)
 	s.Step(`executing "minishift ([^"]*)"`, m.executingCommand)
 	s.Step(`executing "oc ([^"]*)" retrying (\d+) times with wait period of (\d+) seconds$`, m.executingRetryingTimesWithWaitPeriodOfSeconds)
+	s.Step(`([^"]*) of command "minishift ([^"]*)" is equal to "([^"]*)"`, m.commandReturnEquals)
+	s.Step(`([^"]*) of command "minishift ([^"]*)" contains "([^"]*)"`, m.commandReturnContains)
 	s.Step(`executing "oc ([^"]*)`, m.executingOcCommand)
 	s.Step(`([^"]*) should contain ([^"]*)`, m.commandReturnShouldContain)
 	s.Step(`([^"]*) should contain`, m.commandReturnShouldContainContent)
 	s.Step(`([^"]*) should equal ([^"]*)`, m.commandReturnShouldEqual)
 	s.Step(`([^"]*) should equal`, m.commandReturnShouldEqualContent)
 	s.Step(`([^"]*) should be empty`, m.commandReturnShouldBeEmpty)
+	s.Step(`response.?([^"]*)? at "([^"]*)" contains "([^"]*)"`, m.getUrlResponse)
+	s.Step(`JSON config file (contains|does not contain) key "(.*)" with value "(.*)"`, m.configContains)
+	s.Step(`JSON config file (contains|does not contain) key "(.*)"(.*)`, m.configContains)
+	s.Step(`stdout should be valid address`, m.shouldBeValidAddress)
```

Comment:
@hferentschik yes, it means valid IP address. Reason why I am not using should contain is that I was not able to implement a way how to introduce variables like you suggest: #{var}. If I will find solution how to add them, then I agree that should contain is better to use.